src/commands/run.py

```diff
@@ -113,3 +123,22 @@ def remove_done_tasks():
             session.commit()
             Session.remove()
             run_print('Done')
+
+    def get_confirmation(configs, nodes):
+        click.echo("""
+You are about to run: {}\nOver nodes: {}\n
+""".strip().format(', '.join([c.name() for c in configs]),
```

Comment:
There is no point appending a `\n` to the end of the string is you use `strip` to remove it

Reply:
Using a complicated join in the `format` input obscures away what is meant to be displayed. This is particularly the case b/c there are two joins. Instead name the joins in a variable up front:
```
tool_names = ', '.join([c.name() for c in configs])
node_names = ', '.join(nodes)
"""
BLAH
""".format(tool_names, node_names)
```

Reply:
Also can we get a sample output of what this comment looks like? In particular I would like to see what is displayed for a tool group